tools/osbuilder/rootfs-builder/cbl-mariner/rootfs_lib.sh

```diff
@@ -22,69 +22,5 @@ build_rootfs()
 	info "install packages for rootfs"
 	$DNF install ${EXTRA_PKGS} ${PACKAGES}
 
-	# Reduce the image size, for faster TEE memory measurement.
-	local MARINER_REMOVED_PACKAGES=( \
-		"bc" \
-		"bridge-utils" \
-		"bzip2" \
-		"chkconfig" \
-		"cracklib-dicts" \
-		"curl" \
-		"curl-libs" \
-		"cyrus-sasl-lib" \
-		"e2fsprogs" \
-		"expat" \
-		"file" \
-		"findutils" \
-		"gdbm" \
-		"gmp" \
-		"gnupg2" \
-		"gpgme" \
-		"gzip" \
-		"iana-etc" \
-		"iproute" \
-		"iputils" \
-		"krb5" \
-		"libarchive" \
-		"libassuan" \
-		"libdb" \
-		"libksba" \
-		"libpwquality" \
-		"libsolv" \
-		"libssh2" \
-		"libtool" \
-		"libuv" \
-		"libxml2" \
-		"lua-libs" \
-		"mariner-rpm-macros" \
-		"ncurses" \
-		"nghttp2" \
-		"net-tools" \
-		"nettle" \
-		"newt" \
-		"npth" \
-		"openldap" \
-		"openssh-clients" \
-		"openssl" \
-		"pinentry" \
-		"pcre" \
-		"rpm" \
-		"rpm-libs" \
-		"sed" \
-		"shadow-utils" \
-		"sqlite-libs" \
-		"slang" \
-		"sudo" \
-		"tar" \
-		"tzdata" \
-		"xz" \
-	)
-
-	for MARINER_REMOVED_PACKAGE in ${MARINER_REMOVED_PACKAGES[@]}
-	do
-		info "removing package ${MARINER_REMOVED_PACKAGE}"
-		rpm -e "${MARINER_REMOVED_PACKAGE}" --nodeps --root=${ROOTFS_DIR}
-	done
-
 	rm -rf ${ROOTFS_DIR}/usr/share/{bash-completion,cracklib,doc,info,locale,man,misc,pixmaps,terminfo,zoneinfo,zsh}
```

Comment:
Do we know if this is still needed?

Reply:
We should remove these as they have been causing problems for our release process. rootfs binary size should also be ok without it as kata uvm meta-package is smaller than core-packages-base-image.

Reply:
Also we moved most of the Guest packages in a dm-verity protected virtio-blk image. Since those packages are no longer part of an initrd and therefore not measured by the hardware, somewhat larger images are more acceptable nowadays.